http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java

```diff
@@ -113,17 +121,20 @@ private boolean testWithNewClient(int eventLoopSize, int numberOfRequests) {
 
     @DataPoints("EventLoop")
     public static int[] eventLoopValues(){
+        log.warn(() -> "eventLoopValues");
         return new int[]{1, 4};
     }
 
     @DataPoints("ConnectionPool")
     public static int[] connectionsValues(){
         /* Don't use 1 connection Pool of size 1, otherwise test takes too long */
+        log.warn(() -> "connectionsValues");
```

Comment:
Is this log line necessary to keep after the bug is fixed?

Reply:
Had added for debugging , removed them as they are no longer required